src/Components/Footer/Footer.tsx

```diff
@@ -276,6 +277,9 @@ export const Footer: React.FC<FooterProps> = props => {
 
 const PolicyLinks = () => {
   const { CCPARequestComponent, showCCPARequest } = useCCPARequest()
+  const newTermsAndConditionsEnabled = useFeatureFlag(
+    "diamond_new-terms-and-conditions"
+  )
```

Comment:
Why does this simple change need a feature flag?

Reply:
There's a host of changes that are all being placed behind this feature flag so that we can update the terms and conditions across the ecosystem at once. In this case it's useful because the page that "Auction Supplement" links to hasn't been created yet.

Reply:
👍